test/distributed/org/apache/cassandra/distributed/test/accord/AccordIntegrationTest.java

```diff
@@ -166,29 +170,30 @@ public void testRecovery() throws Throwable
 
             awaitAsyncApply(cluster);
 
-            assertRow(cluster, "SELECT * FROM " + keyspace + ".tbl WHERE k=0 AND c=0", 0, 0, 1);
+            //TODO why is this flakey?  Also why does .close hang on Accord?
+            assertRow(cluster, "SELECT * FROM " + keyspace + ".tbl WHERE k=0 AND c=0", 0, 0, 2);
```

Comment:
I hope this is 100% is due to the fact `awaitAsyncApply` does *not* know when accord tx is complete... seeing `0, 0, 1` and `0, 0, 2`... 10s might not be enough anymore due to timeouts being 10s as well?